ethportal-peertest/src/constants.rs

```diff
@@ -1,64 +1,65 @@
 use ethportal_api::{HistoryContentKey, HistoryContentValue};
-use serde_json::json;
+use serde_yaml::Value;
+use std::fs;
 
 /// History HeaderWithProof content key & value
 /// Block #1000010
 pub fn fixture_header_with_proof_1000010() -> (HistoryContentKey, HistoryContentValue) {
+    let yaml_content = fs::read_to_string(
```

Comment:
Bodies of all functions are identical except for the file name.

Create separate function (e.g. `readHistoryContentKeyValue`) that accepts file name as an input and returns `(HistoryContentKey, HistoryContentValue)`.

It can even return `Result<...>` or `Option<...>` but that's up to you.

Reply:
@morph-dev should be good to go now